news/2022-06-01-new-featured-artist-ponchi.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: Ponchi"
+date: 2022-06-01 13:00:00 +0000
+---
+
+**Ponchi** slides into our Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/271/header.jpg)
+
+If you've been following our Featured Artists for the last few months, you're already well aware of this artist's talent.
+
+Alongside [MisomyL](https://osu.ppy.sh/beatmaps/artists/243), **Ponchi** is one of the names behind [MisoilePunch](https://osu.ppy.sh/beatmaps/artists/239), whose music has been featured in countless games like *Dynamix*, *Cytus II*, and *SOUND VOLTEX*! **Ponchi**'s personal works maintain a high-energy rhythm game style with a cute twist enhanced by vocalist *haxchi*'s anime-esque performances.
+
+**7** of these juicy tracks are ripe for the picking from [**Ponchi**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/271)!
+
+Haven't heard **Ponchi**'s sound yet? Now's the perfect time to start. Serenade your ears with this video highlighting two of **Ponchi**'s slow and fast songs while also featuring osu!(lazer)'s new editor metronome.
+
+Oh, and once you're done, check out everything else too:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/271/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Mousou <3 Chuu!
+
+Check out [this speedy set](https://osu.ppy.sh/beatmapsets/1726676) hosted by [Usaha](https://osu.ppy.sh/users/6443117) and seen in the video above!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/271/Songs/Ponchi%20feat.%20haxchi%20-%20Mousou%203%20Chuu!!.mp3" type="audio/mpeg">
+</audio>
+
+### Correct Hearts
+
+Or try [this slower set](https://osu.ppy.sh/beatmapsets/1742463) hosted by [-Aqua](https://osu.ppy.sh/users/7150015) (and also seen in the video)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/271/Songs/Ponchi%20feat.%20haxchi%20-%20Correct%20Hearts.mp3" type="audio/mpeg">
+</audio>
+
+### Anemone
+
+If that first map wasn't fast enough, maybe [this 256 BPM map](https://osu.ppy.sh/beatmapsets/1753715) hosted by [Strategas](https://osu.ppy.sh/users/2971837) will satisfy you.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/271/Songs/Ponchi%20feat.%20haxchi%20-%20Anemone.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+[**Ponchi**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/271) has even more music to latch onto, each with a pre-timed beatmap template to help you out.
+
+In case you missed it, we're now running Featured Artist-themed playlists in osu!(lazer). The top 3 players in each ruleset's playlist at the end of the month will receive a sparkly new profile badge! Read all the details [here](https://osu.ppy.sh/home/news/2022-05-28-new-featured-artist-rameses-b#featured-artists-in-osu!(lazer)).
```

Comment:
```suggestion
In case you missed it, we're now running Featured Artist–themed playlists in osu!(lazer). The top 3 players in each ruleset's playlist at the end of the month will receive a sparkly new profile badge! Read all the details [here](https://osu.ppy.sh/home/news/2022-05-28-new-featured-artist-rameses-b#featured-artists-in-osu!(lazer)).
```